internal/rukpak/source/containers_image_test.go

```diff
@@ -103,7 +108,9 @@ func TestUnpackCacheCheckError(t *testing.T) {
 
 	unpacker := &source.ContainersImageRegistry{
 		BaseCachePath: t.TempDir(),
-		SourceContext: buildPullContext(t, imageTagRef),
+		SourceContextFunc: func(_ logr.Logger) (*types.SystemContext, error) {
+			return buildPullContext(t, imageTagRef)
+		},
```

Comment:
Nit: `_` is not necessary.

```suggestion
		SourceContextFunc: func(logr.Logger) (*types.SystemContext, error) {
			return buildPullContext(t, imageTagRef)
		},
```

Since we are modifying `buildPullContext` anyway we can even make `buildPullContext` to be `buildSourceContextFunc` which returns `func(logr.Logger) (*types.SystemContext, error)` and here we can have something like:

```suggestion
		SourceContextFunc: buildSourceContextFunc(t, imageTagRef),
```

Reply:
Yea you're right, I got lazy. Incorporated this in the [new commit](https://github.com/operator-framework/operator-controller/pull/1322/commits/d7648e61a5825255733702e41826926c26c9b385)